driver/configuration/provider.go

```diff
@@ -19,6 +19,15 @@ const (
 	ForbiddenStrategyErrorType = "forbidden"
 )
 
+// MatchingStrategy defines matching strategy such as Regexp or Glob.
+type MatchingStrategy int
+
+// Possible values.
+const (
+	Regexp MatchingStrategy = iota
```

Comment:
Would it maybe make more sense to have those as string constants `regexp` and `glob` - then it might be easier and read to debug if problems arise :)

Reply:
yes, I agree. changed